src/ZODB/blob.py

```diff
@@ -942,13 +942,14 @@ def remove_committed_dir(path):
                 filename = os.path.join(dirpath, filename)
                 remove_committed(filename)
         shutil.rmtree(path)
-
-    link_or_copy = shutil.copy
+        link_or_copy = shutil.copy
 else:
     remove_committed = os.remove
     remove_committed_dir = shutil.rmtree
-    link_or_copy = os.link
-
+    try:
+        link_or_copy = os.link
+    except AttributeError:    #pragma: no cover
+        link_or_copy = shutil.copy
```

Comment:
Please add a comment indicating why we fall back to `shutil.copy` here, e.g.:
```suggestion
    except AttributeError:    #pragma: no cover
        # FBO termux on Android.
        #  See https://github.com/zopefoundation/ZODB/issues/257
        link_or_copy = shutil.copy
```